src/lib/y2packager/widgets/product_license_content.rb

```diff
@@ -0,0 +1,95 @@
+# ------------------------------------------------------------------------------
+# Copyright (c) 2018 SUSE LLC, All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it under
+# the terms of version 2 of the GNU General Public License as published by the
+# Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+# ------------------------------------------------------------------------------
+
+require "yast"
+require "cwm"
+
+module Y2Packager
+  module Widgets
+    # Widget to show a product's license content
+    class ProductLicenseContent < CWM::CustomWidget
+      # @return [Y2Packager::Product] Product
+      attr_reader :product
+      # @return [String] Language code (en_US, es_ES, etc.).
+      attr_reader :language
+
+      # @param product  [Y2Packager::Product] Product
+      # @param language [String]              Default language (en_US, es_ES, etc.).
+      def initialize(product, language)
+        textdomain "packager"
+        @product = product
+        @language = language
+      end
+
+      # Implement #init content
+      #
+      # @see CWM::AbstractWidget#init
+      def init
+        update_license_text
+      end
+
+      # Return the UI for the widget
+      #
+      # @return [Yast::Term] widget's UI
+      def contents
+        @contents ||= MinWidth(80, license_content)
+      end
+
+      # Translate license content
+      #
+      # @param new_language [String] New language code
+      def translate(new_language)
+        return if language == new_language
+        self.language = new_language
+        update_license_text
+      end
+
+    private
+
+      # @!method language=
```

Comment:
NP: Should the description contain the parameters? `# @!method language=(new_language)`